pkg/reconciler/v1alpha1/pipeline/resources/dag.go

```diff
@@ -21,9 +21,19 @@ import (
 	"strings"
 
 	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
-	errors "github.com/knative/build-pipeline/pkg/errors"
+	"github.com/knative/build-pipeline/pkg/reconciler/v1alpha1/taskrun/list"
```

Comment:
the files in `pipelineRun` reconciler should not be using functions from `taskRun` reconciler. The `list` package should be extracted out

Reply:
@pivotal-nader-ziada hum good point :thinking: I think `taskrun/list` package should be extracted in a upper package (as it really doesn't depend on `taskrun`).
@bobcatfish agreeing ? (I can take care of it :wink:)

Reply:
makes sense @vdemeester